nano/node/backlog_population.cpp

```diff
@@ -13,6 +13,7 @@ nano::backlog_population::backlog_population (const config & config_a, nano::sto
 
 nano::backlog_population::~backlog_population ()
 {
+	stop ();
```

Comment:
This shouldn't be there, while it's not an error, we want to ensure that when node destruction happens all threads have already been stopped.